src/trans.rs

```diff
@@ -719,6 +727,48 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
                 }
             }
 
+            Rvalue::CheckedBinaryOp(ref op, ref left, ref right) => {
+                // TODO: This shouldn't just be a copy BinaryOp above!
+                // We should do some actual _checking_!
```

Comment:
This needs either support for ops that also return the overflow bit, or emulation thereof.